README.md

```diff
@@ -1,21 +1,235 @@
+
+If however your policies are referenced through Xray Watches, follow these steps instead:
+1. Create one or more [Watches](https://www.jfrog.com/confluence/display/JFROG/Configuring+Xray+Watches) on JFrog Xray.
+2. Configure your Watches in the plugin settings: under **Settings (Preferences)** | **Other Settings**, click **JFrog Global Configuration** and go to the **Settings** tab.
+
+# Using the Plugin
+After the JFrog Plugin is installed, a new JFrog panel is added at the bottom of the screen.
+Opening the JFrog panel displays two views:
+
+- The **Local** view displays information about the local code as it is being developed in the IDE.
+  JFrog Xray continuously scans the project's dependencies and source code locally.
+  The information is displayed in the **Local** view.
+- The **CI** view allows the tracking of the code as it is built, tested and scanned by the CI server.
+  It displays information about the status of the build and includes a link to the build log on the CI server.
+
+## The Local View
+The JFrog IntelliJ IDEA Plugin continuously scans your project's dependencies with JFrog Xray and displays this information under the Local view.
+It allows developers to view vulnerability information about their dependencies and source code in their IDE.
+With this information, a developer can make an informed decision on whether to use a component or not before it gets entrenched into the organization’s product.
+
+### Scanning a Project
+Scan your project by clicking the Run Scan ![](readme-resources/run-scan-button.png) button.
+After the scan is done, a list of vulnerable files will appear.
+
+Each descriptor file (like pom.xml in Maven, go.mod in Go, etc.) in the list contains vulnerable dependencies, and each dependency contains the vulnerabilities themselves.
+
+By right-clicking on a dependency line, you can jump to the dependency's declaration in the descriptor file (if it's a direct dependency), or to direct dependencies that depend on the vulnerable component (if any).
+
+![](readme-resources/jump-to-descriptor.png)
+
+By right-clicking on a vulnerability line, you can create an [Ignore Rule](https://www.jfrog.com/confluence/display/JFROG/Ignore+Rules) in Xray.
+
+![](readme-resources/create-ignore-rule.png)
+
+### Viewing Vulnerability Details
+Clicking a vulnerability in the list will open the vulnerability details view.
+This view contains information about the vulnerability, the vulnerable component, fixed versions, impact paths and much more.
+
+![](readme-resources/vuln-details.png)
+![](readme-resources/vuln-impact-graph.png)
+
+### Contextual Analysis
+You can avoid wasting your time fixing risks that are not imposed and reduce false positives with contextual analysis.
+As part of the project's scan, JFrog IntelliJ IDEA Plugin will let you know if vulnerabilities in your dependencies are applicable or not.
```

Comment:
Let's add here a note, describing what you need to do to have the Contexttual Analysis enabled (you need the Adnvaced Security package).